### PR TITLE
chore: make bundle size smaller

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
```
npm run dist
open dist/report.html
```
﻿
image

**Comment by zombieJ** [comment]
清理一下 node_modules，rc-util 其他的 lib 已经清理了。还剩下 icons 的需要单独处理一下代码。

**Comment by zombieJ** [comment]
No more lib
截屏2020-03-28 下午5 03 15

**Comment by zombieJ** [comment]
刚刚好……

截屏2020-03-28 下午6 11 19

**Comment by zombieJ** [comment]
去掉 `@ant-design/icons` es 会让测试挂掉。看看怎么解决。

**Comment by zombieJ** [comment]
回滚一下 icons，会多一个 rc-util/lib/warning。但是影响不大。

File: components/alert/index.tsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 import * as ReactDOM from 'react-dom';
-import CloseOutlined from '@ant-design/icons/lib/icons/CloseOutlined';
-import CheckCircleOutlined from '@ant-design/icons/lib/icons/CheckCircleOutlined';
-import ExclamationCircleOutlined from '@ant-design/icons/lib/icons/ExclamationCircleOutlined';
-import InfoCircleOutlined from '@ant-design/icons/lib/icons/InfoCircleOutlined';
-import CloseCircleOutlined from '@ant-design/icons/lib/icons/CloseCircleOutlined';
-import CheckCircleFilled from '@ant-design/icons/lib/icons/CheckCircleFilled';
-import ExclamationCircleFilled from '@ant-design/icons/lib/icons/ExclamationCircleFilled';
-import InfoCircleFilled from '@ant-design/icons/lib/icons/InfoCircleFilled';
-import CloseCircleFilled from '@ant-design/icons/lib/icons/CloseCircleFilled';
```
Comment (afc163) [inline components/alert/index.tsx]: 上下两个有啥区别？

Comment (zombieJ) [reply components/alert/index.tsx]: 直接写不会被 antd tools 替换成 lib。这个换掉后会导致 typescript 的类型校验 test case 挂掉。回滚了。

**Review by afc163 — APPROVED** [review]

**Pull request merged**